src/Map/index.tsx

```diff
@@ -156,6 +159,19 @@ const Map: React.FC<Props> = (props) => {
     geojsonSource?.setData(sourceJson as FeatureCollection)
   }, [sourceJson])
 
+  // Use to travel to location on card click
+  useEffect(() => {
+    const map = mapRef.current
+    if (firstRender) {
```

Comment:
what happens if the `firstRender` check isn't here?

Reply:
It zooms in a onClickZoomIn on first load which we don't want.